hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/TssConfig.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2023-2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.config.data;
+
+import com.hedera.node.config.NetworkProperty;
+import com.swirlds.config.api.ConfigData;
+import com.swirlds.config.api.ConfigProperty;
+
+/**
+ * Configuration for the TSS service.
+ * @param maxSharesPerNode The maximum number of shares that can be assigned to a node.
+ * @param keyActiveRoster A test-only configuration set to true to key the active roster with TSS key material.
```

Comment:
This is my understanding:
```suggestion
 * @param keyActiveRoster A test-only configuration; set this to true to enable the process that will key the candidate roster with TSS key material.
```
If a roster is active, doesn't it already has the required key material?

Reply:
@mhess-swl I think this configuration is to not wait until upgrade boundary like in production envs. As soon as we get candidate roster we will key it just for tests.

Reply:
That would make sense. If that's the case, would `keyCandidateRoster` be a better name? Not necessary for this PR, but may be helpful to clarify